vipps-ecom-api.md

```diff
@@ -1390,16 +1389,20 @@ See [Timeouts](#timeouts) for details about timeouts.
 
 ## Userinfo
 
-Vipps offers the possibility for merchants to ask for the user's profile
-information as part of the payment flow. This is done by adding a `scope`
+Vipps offers the possibility for merchants to ask for the user's profile information as part of the payment flow. 
+This is done through Vipps Userinfo which 
+You can learn more at the [OIDC Standard](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo).
+
+To enable the posibility to fetch profile information for a user the merchant can add a `scope`
 parameter to the initiate call:
 [`POST:/ecomm/v2/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST).
 
-If the enduser has not already consented to sharing information from Vipps to the merchant the user will be asked for such consent before completing the payment flow. Once the payment flow is completed the merchant can get the profile information from our Userinfo endpoint. The Userinfo endpoint is shared with [Vipps login](https://github.com/vippsas/vipps-login-api) and the merchant needs to have activated Vipps login to use this feature.
+If the enduser has not already consented to sharing information from Vipps to the merchant the user will be asked for 
+such consent before completing the payment flow. Once the payment flow is completed the merchant can get the profile 
```

Comment:
If the enduser has not already consented to sharing information from Vipps to the merchant the user will be asked for any remaining consents before completing the payment flow.

Better phrasing?

Reply:
Also missing a ","